docs/business/alternate_domains.rst

```diff
@@ -0,0 +1,32 @@
+Alternate Domains
+=================
+
+Read the Docs supports a number of custom domains for your convenience.
+
+Subdomain support
+-----------------
+
+Every project has a subdomain that is available to serve its documentation.
+If you go to ``<organization-slug-project-slug>.readthedocs-hosted.com``,
+it should show you the latest version of documentation.
+
+
+CNAME Support
+-------------
+
+If you have your own domain, you can still host with us.
+This requires two steps:
+
+* Add a CNAME record in your DNS that points to our servers ``<organization-slug>.users.readthedocs.com``
```

Comment:
There are more steps here though. We need to discuss set up if your project is private and setup if your project is public.

Reply:
I suppose these two steps are the same for private/public, right?

I left them and I mentioned that after doing these, it's needed to contact the support team.

Reply:
What else ir required if your project is public?